### PR TITLE
Add "secrets" example for service create API docs

### PR DESCRIPTION
**Pull request by thaJeztah**
ping @cyli  @ehazlett  PTAL

### CODE REVIEW

**Comment by thaJeztah** [comment]
Screenshot of the API docs when rendered;

screen shot 2017-05-11 at 17 09 13

**Review by vdemeester — APPROVED** [review]
LGTM  🐮

File: api/swagger.yaml [hunk 1]
```diff
@@ -7612,6 +7612,15 @@ paths:
                         Nameservers: ["8.8.8.8"]
                         Search: ["example.org"]
                         Options: ["timeout:3"]
+                      Secrets:
+                       -
+                         File:
+                          Name: "www.example.org.key"
+                          UID: "33"
+                          GID: "33"
+                          Mode: 384
```
Comment (ehazlett) [inline api/swagger.yaml]: Should this be something like `644` or a mode more common as an example?

Comment (thaJeztah) [inline api/swagger.yaml]: Actually, this was created with `0600` (octal);  which is 384 decimal. The default is `444` (octal), which is `292` decimal

Comment (ehazlett) [inline api/swagger.yaml]: ah right ok - i was looking at it wrong -- thx!

**Comment by ehazlett** [comment]
LGTM

File: api/swagger.yaml (hunk 1 above)
Comment (thaJeztah) [inline api/swagger.yaml]: I agree it's a bit confusing though :smile:

**Comment by thaJeztah** [comment]
Ugh; thought it was formatted ok, let me check

```
15:11:14 api/swagger.yaml
15:11:14   7616:24   error    wrong indentation: expected 24 but found 23  (indentation)
15:11:14   7618:27   error    wrong indentation: expected 27 but found 26  (indentation)
```

**Comment by thaJeztah** [comment]
Should be good now

**Comment by cyli** [comment]
LGTM!

**Pull request merged**